features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Environments/DeploymentOnbording.jsx

```diff
@@ -0,0 +1,312 @@
+import React, { useState } from 'react';
+import 'react-tagsinput/react-tagsinput.css';
+import { FormattedMessage } from 'react-intl';
+import Typography from '@material-ui/core/Typography';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import clsx from 'clsx';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Card from '@material-ui/core/Card';
+import CardContent from '@material-ui/core/CardContent';
+import Tooltip from '@material-ui/core/Tooltip';
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import MenuItem from '@material-ui/core/MenuItem';
+import AddIcon from '@material-ui/icons/Add';
+import CardHeader from '@material-ui/core/CardHeader';
+import PropTypes from 'prop-types';
+import { useAppContext } from 'AppComponents/Shared/AppContext';
+import { useTheme } from '@material-ui/core';
+import Checkbox from '@material-ui/core/Checkbox';
+import RadioButtonUncheckedIcon from '@material-ui/icons/RadioButtonUnchecked';
+import CheckCircleIcon from '@material-ui/icons/CheckCircle';
+
+const useStyles = makeStyles(() => ({
+    root: {
+        minHeight: '480px',
+    },
+    textOptional: {
+        fontFamily: 'sans-serif',
+        fontSize: 'small',
+        color: '#707070',
+        fontWeight: '100',
+    },
+    textRevision: {
+        fontFamily: 'sans-serif',
+        fontSize: '16px',
+        color: '#707070',
+        fontWeight: '800',
+    },
+    textDeploy: {
+        fontFamily: 'sans-serif',
+        fontSize: '26px',
+        color: '#1B3A57',
+        fontWeight: '800',
+    },
+    textDescription: {
+        fontFamily: 'sans-serif',
+        fontSize: '16px',
+        color: '#707070',
+        fontWeight: '400',
+    },
+    descriptionWidth: {
+        minWidth: '550px',
+    },
+    textAlign: {
+        textAlign: 'center',
+    },
+}));
+/**
+ * Renders an Deployment Onboarding
+ * @class Environments
+ * @extends {React.Component}
+ */
+export default function DeploymentOnboarding(props) {
+    const {
+        classes,
+        getVhostHelperText,
+        createDeployRevision,
+        description,
+        setDescription,
+    } = props;
+    const classes1 = useStyles();
+    const theme = useTheme();
+    const { maxCommentLength } = theme.custom;
+    const { settings } = useAppContext();
+    const defaultVhosts = settings.environment.map(
+        (e) => (e.vhosts && e.vhosts.length > 0 ? { env: e.name, vhost: e.vhosts[0].host } : undefined),
+    );
+    const [DescriptionOpen, setDescriptionOpen] = useState(false);
+    const [SelectedEnvironment, setSelectedEnvironment] = useState([]);
```

Comment:
Normally, only React component names start with capital letters, so shall we make this `S` lower case?

Reply:
Fixed https://github.com/wso2/carbon-apimgt/pull/10270/commits/ee20a3b9b54f0df2bafd12c6a3e622938b22935e